_topic_maps/_topic_map.yml

```diff
@@ -554,6 +554,9 @@ Topics:
 - Name: Fedora CoreOS (FCOS) image layering
   File: coreos-layering
   Distros: openshift-origin
+- Name: AWS Local Zone tasks
```

Comment:
I'm unable to find the intro text related to `AWS Local Zone tasks` on this PR. Maybe I'm missing something?